vnext/ReactUWP/Modules/Animated/AnimationDriver.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "AnimationDriver.h"
+
+namespace react {
+  namespace uwp {
+
+    AnimationDriver::AnimationDriver(int64_t id, int64_t animatedValueTag, const Callback& endCallback, const folly::dynamic& config, const std::shared_ptr<NativeAnimatedNodeManager>& manager) :
+      m_id(id), m_animatedValueTag(animatedValueTag), m_endCallback(endCallback), m_config(config), m_manager(manager)
+    {
+      m_iterations = [iterations = config.find("iterations"), end = config.items().end()]() {
+        if (iterations != end)
+        {
+          return iterations.dereference().second.getInt();
+        }
+        return static_cast<int64_t>(1);
+      }();
+    }
+
+    void AnimationDriver::StartAnimation()
+    {
+      const auto [animation, scopedBatch] = MakeAnimation(m_config);
+
+
+      const auto animatedValue = &GetAnimatedValue();
```

Comment:
>& [](start = 33, length = 1)

this seems a little gross, if you're going to turn a ref back into a ptr, it should have just been a pointer to begin with.  If you can't do that, maybe having it return a std::optional is another way to deal with and be able to check for failure